scripts/install-check-tools.sh

```diff
@@ -17,6 +17,12 @@
 echo "Install soci check tools"
 set -eux -o pipefail
 
-curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.53.3
+curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)/bin" v1.53.3
 go install github.com/kunalkushwaha/ltag@v0.2.4
-go install github.com/vbatts/git-validation@v1.2.0
\ No newline at end of file
+go install github.com/vbatts/git-validation@v1.2.0
+
+scversion="v0.9.0"
+arch=$(uname -p)
+wget -qO- "https://github.com/koalaman/shellcheck/releases/download/${scversion?}/shellcheck-${scversion?}.linux.${arch?}.tar.xz" \
+    | tar -xJv
+shellcheck --version
```

Comment:
Needs a little more thought. Runner comes with shellcheck preinstalled so later in CI the wrong binary is being used.

Reply:
Can we just use the runner version?